doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+Account A has cancelled all 2 of their orders.
+TRANSACTIONS EXECUTED (first-to-last):
+  * Cancel all ASKS for Account A: 0xbd77261240ede717d95af21d1887b7292271a8675fe8ced54aa100d998e2edc5
+  * Cancel all BIDS for Account A:: 0x207b2cdeea95fa60549301d77e72baa38fe99346893d771ed59ffe9df8c0167c
+CURRENT BEST PRICE LEVELS:
+There is no tETH being bought or sold right now!
+```
+
+This one is straightforward, but it's worth nothing that there are no longer orders on the book unlike in the step above. That's because in this case, all of the liquidity in the order book has been cancelled by our cancelling all of Account A's orders (since Account A's orders were all the orders on the book).
+
+#### Step 7: Place Multiple Competitive Limit Orders (as Account A)
+
+```
+Press enter to place competitive limit orders (top-of-book) with Account A.
+Account A has created multiple competitive limit orders!
+TRANSACTIONS EXECUTED (first-to-last):
+  * Place limit BID/BUY order (100 lots) (1000 ticks/lot): 0xaa5eaf815072342645a85dbd7e40e84d999d44fae484440832474ccc7927cc18
+  * Place limit ASK/SELL order (100 lots) (2000 ticks/lot): 0x7ec72c68aef04221eba37e83041440c2ef0e9bfe68bfc70b9e73580508177237
+  * Place limit BID/BUY order (200 lots) (1001 ticks/lot): 0x463316baead724ff08021e2e8d59fbeefcffffb19e6ea026d167005e69168268
+  * Place limit ASK/SELL order (200 lots) (1999 ticks/lot): 0xd19cacbe7b6a2496e91fd17fd359e228f1b5bd9d6ac9c4f0db706e8ad24a9939
+  * Place limit BID/BUY order (300 lots) (1002 ticks/lot): 0xbb9b637d01b6209c13d760e7a1b9dbd4fcdd05a724b5ed0e4c50463be1566d3d
+  * Place limit ASK/SELL order (300 lots) (1998 ticks/lot): 0x7ad8c60956edaee5c3d4a8d355c015a5c5776f79ec19310f8d91b7a1ba3e159a
+  * Place limit BID/BUY order (400 lots) (1003 ticks/lot): 0x64a1e2bc60a50ad2a44fdadb9aaa4f75a03a1061af5ebe926a79619ba4acdab8
+  * Place limit ASK/SELL order (400 lots) (1997 ticks/lot): 0x2acba7786094cbaa5f3551f9489847e33f9d114734ff9f35bdc815488cb0c8ea
+  * Place limit BID/BUY order (500 lots) (1004 ticks/lot): 0x2038221c4fa6488f3645fe6722ef1269f19ecd61d7e4352a8767cb12106ba3d8
+  * Place limit ASK/SELL order (500 lots) (1996 ticks/lot): 0xe4754a6069c76275a0d8ea3933825160abb9bddf78a3cee7f27d798e479df337
+CURRENT BEST PRICE LEVELS:
+  * Highest BID/BUY @ 1004 ticks/lot, 500 lots
+  * Lowest ASK/SELL @ 1996 ticks/lot, 500 lots
+```
+
+Here you can see that Account A is placing multiple limit orders in each direction. Every order "beats" the last, for example the bid at 1001 ticks per lot across 200 lots "beats" the bid at 1000 ticks per lot across 100 lots. The highest price the base asset (tETH) is being bought for right now is 1004 ticks/lot with 500 lots at that price. Likewise the lowest price the base asset (tETH) is being sold for right now is 1996 ticks/lot with 500 lots at that price.
+
+We'll see the order of these limit orders clearer in the next step, where Account B places spread-crossing limit orders.
```

Comment:
Change "order of these" to "sequence of these" since the word order comes a bit later and sounds awkward